libraries/rush-lib/src/schemas/subspace.schema.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Rush subspace config file.",
+  "description": "The configuration file for enabling the subspaces feature in rush.",
+  "type": "object",
+
+  "properties": {
+    "$schema": {
+      "description": "Part of the JSON Schema standard, this optional keyword declares the URL of the schema that the file conforms to. Editors may download the schema and use it to perform syntax highlighting.",
+      "type": "string"
+    },
+    "enabled": {
```

Comment:
For a casual observer (who upgraded Rush but has never heard of subspaces), we need to clearly indicate that:
- This feature is still "EXPERIMENTAL" 
- The implementation is incomplete and not yet usable
- Which setting is used to opt-in to the experiment

During code reviews, it will need to be easy for us to prove that the experimental algorithms will not be invoked unless the user has opted-in to the experiment.  (Because for any code that is reachable in the opted-out state, we would need to do careful testing to ensure no regressions are introduced.)